themes/bootstrap3/less/vendor/bootstrap/alerts.less

```diff
@@ -34,6 +34,19 @@
   }
 }
 
+// General purpose system message style
+.system-message {
+  background-color: @state-warning-bg;
+  padding-top: 0.6em;
+  padding-bottom: 0.6em;
+  text-align: center;
+  z-index: 1;
+  @media (max-width: @grid-float-breakpoint) {
+    position: relative;
+    top: @navbar-height;
+  }
+}
```

Comment:
The mobile styling overlaps the content due to the use of `top: 50px;`. Would it make things simpler and clearer if we use the stylings from [`.alert.alert-warning`](https://getbootstrap.com/docs/3.4/components/#alerts-examples)?

![image](https://user-images.githubusercontent.com/451107/157059410-e98dbc93-8c7b-4dc9-8908-bc04540f4fea.png)

Reply:
I can try that one too!